Sources/MessagingPush/MessagingPushImplementation.swift

```diff
@@ -8,167 +8,52 @@ import UserNotifications
 
 internal class MessagingPushImplementation: MessagingPushInstance {
     let siteId: SiteId
-    let profileStore: ProfileStore
-    let backgroundQueue: Queue
-    var globalDataStore: GlobalDataStore
     let logger: Logger
-    let sdkConfig: SdkConfig
     let jsonAdapter: JsonAdapter
-    let deviceAttributesProvider: DeviceAttributesProvider
-    let dateUtil: DateUtil
```

Comment:
`sdkConfig`, `devceAttrbutesProvider`. `globalDataStore` were some of the properties that were seen in customer logs, who reported crashes in their app. I hope this helps in overcoming those issues

Reply:
Agreed!